.github/ISSUE_TEMPLATE/research-issue-template-.md

```diff
@@ -0,0 +1,51 @@
+---
+name: 'Research issue template'
+about: Issue template for creating new research issues
+title: ''
+labels: research
+assignees: ''
+
+---
+
+Description:
+ 
+
+AC: 
+
+- [ ] [Draft research plan]/[user interviews]/[synthesis] is complete. 
+
+Proposed Tasks: 
+
+Research plan
+- [ ] Create a draft research plan that addresses the following epics - [...]
+   - [ ] List research goals
+   - [ ] Determine method(s)
+   - [ ] Develop recruitment criteria
+   - [ ] Draft timeline with activities, timeline, and team participation
+   - [ ] Document ethics, privacy concerns
+   - [ ] Draft expected outcome with actionable learnings including outputs (diagrams, documents, etc.)
+   - [ ] Write interview guide
+- [ ] Determine logistics for conducting interviews
+   - [ ] Schedule interviews and confirm participation
+   - [ ] Create email language template (if applicable)
+   - [ ] Create template for recording interview notes
+   - [ ] Dry run
+
+Conduct Interviews
+ - [ ] Email privacy statement to participants 1 day in advance
+ - [ ] Record Video if applicable
+ - [ ] Record verbatim notes in the notes template 
+ 
+Synthesis
+- [ ] Gather all notes and fill in gaps with video recording
+- [ ] Delete video recording once notes are complied
+- [ ] Deidentify data 
+- [ ] Perform topline synthesis
+- [ ] Synthesis workshop with PO/stakeholders
+- [ ] Document synthesis - Scope (research goal(s)), High-level findings, Actionable learnings, Issues that have to be added to Zenhub, Takeways for future research
+- [ ] Document 'What we focussed on', 'Key Takeaways', and 'Opportuntiies for future research' in Wiki.
```

Comment:
fixing some typos

`Document 'What we focused on', 'Key Takeaways', and 'Opportunities for future research' in Wiki.`